src/ts/app/view/viewport/handler/mouse.ts

```diff
@@ -117,6 +127,105 @@ export class MouseHandler {
         listenOnce(document, 'mouseup', this.shiftOnMouseUp)
     }
 
+    selectionHandlePressed = () => {
+        console.log('selection handle pressed!')
+        // before anything else, disable the camera
+        this.viewport.cameraIsLocked = true
+
+        // record the starting positions of selected landmarks
+        this.dragStartPositions = this.viewport.selectedLandmarks
+            .map(lm => [lm.index, lm.point.clone()])
+
+        // start listening for dragging selection handle
+        document.addEventListener('mousemove', this.selectionHandleOnDrag)
+        listenOnce(document, 'mouseup', this.selectionHandleOnMouseUp)
+    }
+
+    selectionBoxPressed = (event: MouseEvent) => {
+        console.log('selection box pressed!')
+        // before anything else, disable the camera
+        this.viewport.cameraIsLocked = true
+
+        // record the starting positions of selected landmarks
+        this.positionLmDrag.copy(new THREE.Vector2(event.clientX, event.clientY))
+        this.dragStartPositions = this.viewport.selectedLandmarks
+            .map(lm => [lm.index, lm.point.clone()])
+
+        // start listening for dragging selection box
+        document.addEventListener('mousemove', this.landmarkOrSelectionBoxOnDrag)
+        listenOnce(document, 'mouseup', this.selectionBoxOnMouseUp)
+    }
+
+    rotationHandlePressed = () => {
+        console.log('rotation handle pressed!')
+        // before anything else, disable the camera
+        this.viewport.cameraIsLocked = true
+        this.viewport.activateRotationCircle()
+
+        this.viewport.clearCanvas()
+        this.viewport.drawRotationCircle()
+
+        // record the starting positions of selected landmarks
+        this.dragStartPositions = this.viewport.selectedLandmarks
+            .map(lm => [lm.index, lm.point.clone()])
+
+        // start listening for dragging rotation handle
+        document.addEventListener('mousemove', this.rotationHandleOnDrag)
+        listenOnce(document, 'mouseup', this.rotationHandleOnMouseUp)
+    }
+
+    mouseDownOnSelectionHandle = () => {
+        const md = this.onMouseDownPosition
+        const sb = this.viewport.selectionBox
+        const hr = sb.handleRadius
+        if (sb.minPosition.x === -1 && sb.minPosition.y === -1 && sb.maxPosition.x === -1 && sb.maxPosition.y === -1) {
+            console.log("selection box inactive!")
+            return false
+        }
+        if (md.x <= sb.minPosition.x + hr && md.x >= sb.minPosition.x - hr && md.y <= sb.minPosition.y + hr && md.y >= sb.minPosition.y - hr) {
```

Comment:
@gabrielcowley might be worth adding a small padding around the selection box handles (and the rotation handle) so that if the user misses by a pixel or so it still triggers the behaviour. Literally one pixel boundary would probably do it? No change to the UI, but the mouse grab zone grows by 1.

(I've just found sometimes I'm not being quite precise enough to get it at current setting)